tests/system/projects/test_project.py

```diff
@@ -641,6 +645,53 @@ def _test_remote_pipeline_from_github(
                 nuclio_function_url, notification_steps
             )
 
+    def test_remote_pipeline_with_workflow_runner_node_selector(self):
+        project_name = "rmtpipe-kfp-github"
+        self.custom_project_names_to_delete.append(project_name)
+
+        workflow_name = "neflow"
+        workflow_runner_name = f"workflow-runner-{workflow_name}"
+        runner_node_selector = {"kubernetes.io/arch": "amd64"}
+        project_default_function_node_selector = {"kubernetes.io/os": "linux"}
+
+        project = self._load_remote_pipeline_project(name=project_name)
+        project.spec.default_function_node_selector = (
+            project_default_function_node_selector
+        )
+        project.save()
+
+        project.run(
+            workflow_name,
+            watch=False,
+            local=False,
+            engine="remote",
+            workflow_runner_node_selector=runner_node_selector,
+        )
+        runner_run_result = project.list_runs(name=workflow_runner_name)[0]
+        assert runner_run_result["spec"]["node_selector"] == {
+            **project_default_function_node_selector,
+            **runner_node_selector,
+        }
+
+        # Test scheduled workflow
+        schedule = "*/30 * * * *"
```

Comment:
I think you need some schedule crontab that will never run so we won't have the schedule invoked twice by accident